Session.js

```diff
@@ -1391,15 +1405,35 @@ Session.prototype = {
 			return getDocumentActiveElement();
 		}
 		else {
-			return this._post('element/active').then(function (element) {
+			var activeFunc = self.capabilities.useGetForActiveElement ?
+				self._get :
+				self._post;
+			return activeFunc.call(this, 'element/active').then(function (element) {
 				if (element) {
+					console.log('there an element');
 					return new Element(element, self);
 				}
 				// The driver will return `null` if the active element is the body element; for consistency with how
 				// the DOM `document.activeElement` property works, we’ll diverge and always return an element
 				else {
 					return getDocumentActiveElement();
 				}
+			}).catch(function(error) {
+				if (error.detail.error === 'unknown command') {
```

Comment:
This should also check if we've already set `self.useGetForActiveElement`. If we get to this point and that flag is already set, we don't need to set it again and retry the operation.

Reply:
Fixed